src/jvm/test/org/jetbrains/kotlinx/lincheck_test/util/TestUtils.kt

```diff
@@ -79,6 +85,9 @@ private val TEST_EXECUTION_TRACE_ELEMENT_REGEX = listOf(
 ).joinToString(separator = ")|(", prefix = "(", postfix = ")").toRegex()
 
 private val LINE_NUMBER_REGEX = Regex(":(\\d+)")
+private val INNER_CLASS_INDEX_REGEX = Regex("#(\\d+)")
```

Comment:
Are we sure we want to ignore object numbers? It can then accidentally create "collisions" betwen different objects in the trace.

Reply:
Yes, they are synthetic and created by JVM itself. I hope they have no errors here on their side.